atlas/templates/Jenkinsfile.gotmpl

```diff
@@ -31,21 +37,24 @@ pipeline {
       steps {
         withDockerRegistry([credentialsId: "<insert-the-creds-id>", url: ""]) {
           dir("$DIRECTORY") {
-            sh "make docker push"
+            // The shell started by Jenkins should inherit the env vars defined above.
+            // Since Makefile.common uses ?= to set GIT_COMMIT and IMAGE_VERSION, then the values
+            // set above should override the defaults from Makefile.common.
+            sh "make docker"
           }
         }
       }
     }
     stage("Push") {
       when {
-        branch "master"
+        // This check will work regardless of whether the merge build runs off of the master
+        // or develop branches.
+        expression{ !isPrBuild() }
```

Comment:
This is probably okay. Neither case considered integration tests that require images be available for k8s to pull.